gensim/models/wrappers/wordrank.py

```diff
@@ -89,25 +88,25 @@ def train(cls, wr_path, corpus_file, out_name, size=100, window=15, symmetric=1,
         """
 
         # prepare training data (cooccurrence matrix and vocab)
-        model_dir = join(wr_path, out_name)
-        meta_dir = join(model_dir, 'meta')
+        model_dir = os.path.join(wr_path, out_name)
+        meta_dir = os.path.join(model_dir, 'meta')
         os.makedirs(meta_dir)
         logger.info("Dumped data will be stored in '%s'", model_dir)
-        copyfile(corpus_file, join(meta_dir, corpus_file.split('/')[-1]))
+        copyfile(corpus_file, os.path.join(meta_dir, corpus_file.split('/')[-1]))
 
-        vocab_file = join(meta_dir, 'vocab.txt')
-        temp_vocab_file = join(meta_dir, 'tempvocab.txt')
-        cooccurrence_file = join(meta_dir, 'cooccurrence')
-        cooccurrence_shuf_file = join(meta_dir, 'wiki.toy')
-        meta_file = join(meta_dir, 'meta')
+        vocab_file = os.path.join(meta_dir, 'vocab.txt')
+        temp_vocab_file = os.path.join(meta_dir, 'tempvocab.txt')
+        cooccurrence_file = os.path.join(meta_dir, 'cooccurrence')
+        cooccurrence_shuf_file = os.path.join(meta_dir, 'wiki.toy')
+        meta_file = os.path.join(meta_dir, 'meta')
 
-        cmd_vocab_count = [join(wr_path, 'glove', 'vocab_count'), '-min-count', str(min_count), '-max-vocab', str(max_vocab_size)]
-        cmd_cooccurence_count = [join(wr_path, 'glove', 'cooccur'), '-memory', str(memory), '-vocab-file', temp_vocab_file, '-window-size', str(window), '-symmetric', str(symmetric)]
-        cmd_shuffle_cooccurences = [join(wr_path, 'glove', 'shuffle'), '-memory', str(memory)]
+        cmd_vocab_count = [os.path.join(wr_path, 'glove', 'vocab_count'), '-min-count', str(min_count), '-max-vocab', str(max_vocab_size)]
+        cmd_cooccurence_count = [os.path.join(wr_path, 'glove', 'cooccur'), '-memory', str(memory), '-vocab-file', temp_vocab_file, '-window-size', str(window), '-symmetric', str(symmetric)]
+        cmd_shuffle_cooccurences = [os.path.join(wr_path, 'glove', 'shuffle'), '-memory', str(memory)]
         cmd_del_vocab_freq = ['cut', '-d', " ", '-f', '1', temp_vocab_file]
 
         commands = [cmd_vocab_count, cmd_cooccurence_count, cmd_shuffle_cooccurences]
-        input_fnames = [join(meta_dir, corpus_file.split('/')[-1]), join(meta_dir, corpus_file.split('/')[-1]), cooccurrence_file]
+        input_fnames = [os.path.join(meta_dir, os.path.split(corpus_file)[-1]), os.path.join(meta_dir, os.path.split(corpus_file)[-1]), cooccurrence_file]
```

Comment:
This line is a little hard to navigate -- any way to restructure the logic to make it more readable? Maybe factor out some of the arguments into separate lines?